notices.json

```diff
@@ -37,21 +37,21 @@
     }
   },
   {
-    "id": "server_upgrade_v5.39",
+    "id": "server_upgrade_v6.00",
     "conditions": {
       "audience": "sysadmin",
       "clientType": "all",
-      "serverVersion": ["<5.39"],
+      "serverVersion": ["<6.00"],
       "instanceType": "onprem",
-      "displayDate": ">= 2021-09-21T00:00:00Z"
+      "displayDate": ">= 2021-10-15T00:00:00Z"
```

Comment:
0/5, maybe Monday 18th is better than a Monday.

Reply:
+1